text/src/autogluon/text/automm/predictor.py

```diff
@@ -551,11 +555,24 @@ def _fit(
             fast_dev_run=config.env.fast_dev_run,
             val_check_interval=config.optimization.val_check_interval,
         )
-        trainer.fit(
-            task,
-            datamodule=train_dm,
-            ckpt_path=self._ckpt_path,  # this is to resume training that was broken accidentally
-        )
+
+        with warnings.catch_warnings():
+            warnings.filterwarnings(
+                "ignore",
+                ".*does not have many workers which may be a bottleneck. "
+                "Consider increasing the value of the `num_workers` argument` "
+                "\(try 16 which is the number of cpus on this machine\) in the "
```

Comment:
you hard-coded 16 into the message, not all machines will have 16 cores so this won't be filtered.

Reply:
Thanks. Didn't notice this.